commands

```diff
@@ -180,6 +219,35 @@ case "$1" in
     check_container
     docker logs dokku-mongodb | tail -n 100
     ;;
+  mongodb:restore) 
+    verify_min_args 2 "$@"
+    check_container
+
+    fileOrDir="${3%/}"
+    # does file or folder exist?
+    if [[ ! -d "$fileOrDir" && ! -f "$fileOrDir" ]]; then
+      fail "$fileOrDir doesn't exist"
+    fi
+    
+    if [[ $fileOrDir == *"tar.gz" ]]; then 
+      tempdir="/tmp/mongodump"
+      rm -rf "$tempdir"
```

Comment:
Maybe it would be better to have some random string (or timestamp or something) in the tmp folder name like /tmp/mongodump/date.random . The folder would need to be deleted after successful restore. My thinking behind this is - imagine if someone was simultaneously restoring from two backups at the same time (why would you do that :) .. someone would probably try it though). This would blow away the first backup mid-restore when you start the second.